source/frontend/src/features/properties/map/acquisition/detail/AcquisitionFileTabs.tsx

```diff
@@ -33,6 +34,9 @@ export const AcquisitionFileTabs: React.FC<IAcquisitionFileTabsProps> = ({
 }) => {
   const tabViews: TabFileView[] = [];
   const { hasClaim } = useKeycloakWrapper();
+  const [activeTab, setActiveTab] = useState<FileTabType>(defaultTab);
```

Comment:
Concerned about this. We should not be putting state in non-container objects and 100% shouldn't be duplicating state from a parent like this. Can you explain why this is necessary? This seems problematic.

Reply:
This was already in place but placed at the bottom of the file, I move it to the top to be alongside other declarations. Would rather create a tech debt to analyze later.